lib/commands/create.js

```diff
@@ -72,7 +72,7 @@ function create(context) {
         }
       }
     }).then(function(build) {
-      request.get(build.stream_url).pipe(process.stderr);
```

Comment:
The shorter one, I guess it's just now that the decision was made to choose `output_stream_url` instead.